### ISSUE TITLE
Auto-Install Plugins using Easy Engine 

### ISSUE THREAD

**Issue by rahul286** [rtCamp/easyengine#536]
http://community.rtcamp.com/t/auto-install-plugins-using-easy-engine/2211

**Comment by ddelaey** [comment]
+1 Would be very handy to make own 'standard' installs.

**Comment by Kuznetsof** [comment]
+1 Yep would like to have some command to install list of plugins and a few themes

**Comment by rippler** [comment]
Please don't forget auto-uninstall as well. Aksimet and Dolly are going on my nerves.

**Comment by Kuznetsof** [comment]
@rippler +5 ' Aksimet and Dolly are going on my nerves'

**Comment by irazasyed** [comment]
+1 for auto-install and uninstall as well.

**Comment by xkidro** [comment]
+1 This would be quite awesome

**Comment by andrewklimek** [comment]
+1  I think that would put an extra E in Easy

**Comment by michelkusters** [comment]
+1

**Comment by aronwp** [comment]
would be nice to have it in ee but I just use wp-cli in a script. Works for my team. We setup a script template and adjust the details based on the site/project. It does everything to setup a site. Installs themes and plugins, creates child theme, removes dolly and akismet plugins, hello world posts, default widgets, sets time zone, creates navigation and assigns it. Basically everything wp-cli does.

**Comment by bigwilliam** [comment]
I agree with aronwp - wp-cli already handles this feature, and I think adding it to EE would just be re-inventing the wheel. I feel like there are other features which should take precedence over this.

However, it would be nice if EE had some hooks where we could place our own custom scripts to be run during each install. We could then put our scripts in that folder and have them be run for each new EE site. One of those scripts could be to install a specific set of plugins.

**Comment by Kuznetsof** [comment]
Thanks for pointing out @aronwp

**Comment by Kuznetsof** [comment]
The problem with http://wp-cli.org/ is after spending some time it looks like 
an easy way to install single plugins, but defiantly it should be an Easier (human) way of
how to pack plugins sets. 

Here's what i want to see > place online with an option to create an account with
repositories, within repos i would like all plugins i need updated to the latest version.

Once i do: 

ee site create example.com --wp [url to my repo link that works like key]
 
This can be FREE service, the Gravy comes if i can have all my keys for premium plugins
in place once in cpanel and not required anymore.

Amen.

**Comment by pingram3541** [comment]
Looking at the latest ee.conf I see:

```
### Where external (third-party) plugins are loaded from
# plugin_dir = /var/lib/ee/plugins/
```

So is this something different or related to this thread and just not implemented yet?  I created a plugins folder and dropped a few in but not sure if they need to be zipped or same dir structure as the plugins folder or if this is something else entirely.  I couldn't find clarity in the documentation but when I ran a site install they did not copy over so I removed them until I can find out more.  For now a basic cp command works =)

**Comment by codente** [comment]
This would be a killer feature.  And even better, something like blueprints where we could create different sets and select which blueprint/set to install via a short command
